lib/membrane/rtp/ssrc_router.ex

```diff
@@ -10,6 +10,21 @@ defmodule Membrane.RTP.SSRCRouter do
 
   alias Membrane.{RTP, RTCPEvent}
 
+  def_options rtcp_pad_mapper: [
+                type: :function,
+                spec:
+                  (rtp_pad_id :: any() ->
+                     rtcp_pad_id :: any()),
+                default: &__MODULE__.default_rtcp_pad_mapper/1,
+                description: """
+                A function that given an id of RTP's input pad from which came packets with certain ssrcs returns
+                an id of an input pad to which RTCP events with mentioned ssrcs should be forwarded.
+
+                By default incoming RTCP events wil be forwarded to an input pad of id `{:rtcp, id}` where id
+                belongs to an original RTP input pad from which packets with RTCP event's ssrc came.
+                """
+              ]
```

Comment:
I don't think anybody ever uses this option, let's hardcode that to `{:rtcp, id}` and describe that in the moduledoc